package.json

```diff
@@ -7,16 +7,16 @@
   "author": "Michał Pierzchała <thymikee@gmail.com>",
   "license": "BSD-3",
   "dependencies": {
-    "jest-zone-patch": "^0.0.7",
-    "jsdom": "^11.3.0",
-    "ts-jest": "^21.0.0"
+    "jest-zone-patch": "^0.0.8",
+    "jsdom": "11.5.1",
+    "ts-jest": "21.2.4"
   },
   "devDependencies": {
-    "jest": "^21.0.1",
-    "typescript": "^2.3.0"
+    "jest": "21.2.1",
```

Comment:
I prefer `^21.2.1`, we have a lockfile

Reply:
do you mean you prefer `^21.0.1` ?

Reply:
Doesn't really matter, because it will resolve to 21.x.x. I'd go with `^21.2.1` if I had to choose :)

Reply:
then I'll keep this unchanged and only commit to change the version in `peerDependencies` back to `^21.0.1`